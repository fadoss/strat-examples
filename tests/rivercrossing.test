games/rivercrossing

srew init using allCE ; (match (s(right) w(right) g(right) c(right))) .
