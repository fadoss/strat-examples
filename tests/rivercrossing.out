==========================================
srewrite in RIVER-CROSSING-STRAT : init using allCE ; match s(right) w(right)
    g(right) c(right) .

Solution 1
rewrites: 151
result Group: s(right) w(right) g(right) c(right)

No more solutions.
rewrites: 167
Bye.
